### PR TITLE
adding files

### PR DESCRIPTION
**Pull request by dmcassel**
Adding files as described in issue #1.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+* **Configuration** - If you can provide part of the configuration that leads 
+to a problem, that allows us to duplicate it
+* **Suggest a Fix** - if you can't fix the bug yourself, perhaps you can point to what might be
+  causing the problem (line of code or commit)
+
+### Submitting a Pull Request
+
+#### Fork mlsound
+
+Fork the project [on GitHub](https://github.com/marklogic/mlsound/fork) and clone
```
Comment (grtjn) [inline CONTRIBUTING.md]: marklogic -> miguelrgonzalez

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+your copy.
+
+```sh
+$ git clone git@github.com:username/mlsound.git
+$ cd mlsound
+$ git remote add upstream git://github.com/marklogic/mlsound.git
```
Comment (grtjn) [inline CONTRIBUTING.md]: marklogic -> miguelrgonzalez

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    git branch -D my-feature-branch
+    ```
+
+* Update your master with the latest upstream version:
+
+    ```shell
+    git pull --ff upstream master
+    ```
+
+[issue tracker]: https://github.com/marklogic/mlsound/issues
```
Comment (grtjn) [inline CONTRIBUTING.md]: marklogic -> miguelrgonzalez

**Comment by grtjn** [comment]
Miguel might prefer a different license file? ;-)

**Comment by dmcassel** [comment]
Thanks @grtjn, I fixed the GitHub URL references. For the license file, I went with the standard Apache license used by most MarkLogic-related projects.

**Pull request merged**